fiatconnect-api.md

```diff
@@ -1796,6 +1807,62 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+#### 7.3.2.2. `MobileMoney`
+
+Most of the mobile money's providers require only the phone number to process a transaction.  So, the best approach to make this schema general, is to add the *operator*.
+`Operator` represents the name of the mobile operator and `mobile` the phone number of the end-users. The property `mobile` should follow the [International format E.164 from ITU-T](https://en.wikipedia.org/wiki/E.164) (i.e., +14155552671 for US).
+
+```
+{
+  accountName: `string`,
+  institutionName: `string`,
+  mobile: `string`,
+  operator: `SupportedOperatorEnum`,
+  fiatAccountType: `FiatAccountTypeEnum.MobileMoney`
+}
+```
+
+##### 7.3.2.2.1. `SupportedOperatorEnum`
+
+Depending on the `allowedValues` field for `operator` in each country, the client SHOULD impose restrictions on the type of data the user can provide for the `operator` field. This data should be part of the `SupportedOperatorEnum` provided for each country. Below you can find a list of mobile money provider for each country 
+(PS: New mobile money providers can be added).
+
+###### 7.3.2.2.1.1. Côte d'Ivoire
```

Comment:
Rather than having a list for each country here, with providers duplicated between them, can we just have a single `SupportedOperatorEnum` list that lists _all_ potential providers? FiatConnect API implementors will be able to choose whatever they want from that list depending on the country they're in, and interpret the user's selection however they please depending on their country. For example, just a single list, with the mappings beneath it:

```
[
  `ORANGE`,
  `MOOV`,
  `MTN`,
  `WAVE`
]
```

- `ORANGE` - [Orange Money](https://en.wikipedia.org/wiki/Orange_Money)
- `MOOV` - [Moov Money](https://www.moov-africa.ci/moov-money/)
- `MTN` - [Momo](https://www.mtn.ci/vos/depot-et-retrait-momo/)
- `WAVE` - [Wave](https://www.wave.com/fr/)

Reply:
I am ok with you